ember/app/routes/tracking/redirect.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  redirect(model) {
+    this.transitionTo('tracking.details', model);
```

Comment:
the `model` seems to contain e.g. `user_ids: '1,2,3'`, but nothing more. feeding that model into the transition like that will cause the `model()` hook of `tracking.details` to not be called and use the model passed into the transition instead. what we need to do to prevent this is pass on the plain `user_ids` value to the transition instead:

```js
  redirect({ user_ids }) {
    this.transitionTo('tracking.details', user_ids);
  },
```